src/memray/reporters/assets/flamegraph.js

```diff
@@ -42,9 +42,29 @@ function packedDataToTree(packedData) {
   return root;
 }
 
+function initTrees(packedData) {
+  const { strings, nodes, inverted_import_system_nodes, unique_threads } =
+    packedData;
+  flamegraphData = packedDataToTree({
+    strings: strings,
+    nodes: nodes,
+    unique_threads: unique_threads,
+  });
+
+  invertedImportSystemData = inverted
+    ? packedDataToTree({
+        strings: strings,
+        nodes: inverted_import_system_nodes,
+        unique_threads: unique_threads,
+      })
+    : {};
```

Comment:
Let's do `null` here instead of an empty object. That's a better indicator for the particular sort of invalid that this is.
```suggestion
    : null;
```